src/components/common/LoadingPage.js

```diff
@@ -10,11 +10,38 @@ import "styles/Loading.scss";
 		textPadding: string representing padding to the left of the text, i.e. distance from the img (give px units)
 */
 
-const Loading = props => (
-  <div className="Loading">
-    <div className="Loading-title">Loading</div>
-    <RingLoader color={"#171124"} size={250} loading={true} />
-  </div>
-);
+class Loading extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showText: false,
+      timer: 0,
+    };
+  }
+  componentDidMount = () => {
+    this.setState.timer = setTimeout(() => {
+      this.setState({ showText: true });
+    }, 2000);
+  };
+
+  componentWillUnmount = () => {
+    clearTimeout(this.setState.timer);
+  };
+
+  render = () => {
+    return (
+      <div className="Loading">
+        <div className="Loading-title">Loading</div>
+        <RingLoader color={"#171124"} size={250} loading={true} />
+        {this.state.showText && (
+          <p className="Loading-page-text" style={{ color: "white" }}>
```

Comment:
Right now, the `Loading-page-text` class isn't defined. However, you could add it (in `src/styles/Loading.scss`) and then apply CSS styles that way, instead of using the style tag.

Also, right now you have `color: white`, but we actually don't use white on the loading page, we use an off-white. In the `.scss` file, you can set `color: $off-white` (similar to the line for `.Loading-title`).

I'd also maybe add a bit of margin/padding to the top and the sides: it's a bit too close to the spinning loader, and if you shrink the viewport (i.e. make the browser smaller) the text sticks a bit too close to the sides.